### ISSUE TITLE
Problem with Rietveld Refinement of Sample Fluroapatite data set

### ISSUE THREAD

**Issue by spinglery** [AdvancedPhotonSource/GSAS-II#56]
Dear all,
I try to reproduce the Fluroapatite example of a Rietveld refinement. I run the latest version of GSAS-II (5793) on a Windows11 Laptop. Already after the background correction, my refinement looks quite different than in the tutorial (see screenshot). I did notice in the Python logfile that the GitPython is not installed (see logfile). I tried to install it according to https://gitpython.readthedocs.io/en/stable/intro.html, but this failed. I have also attached the gpx file.
Thanks a lot for your help!
Best regards
Bernhard
[GSAS-II.txt](https://github.com/user-attachments/files/16589292/GSAS-II.txt)
![Screenshot 2024-08-12 171655](https://github.com/user-attachments/assets/4254d3c6-42d6-4e8b-9497-b8ccb9c05142)
[Fluroapatite.zip](https://github.com/user-attachments/files/16589307/Fluroapatite.zip)

**Comment by briantoby** [comment]
The gitpython warning is there because you are using an older version of GSAS-II that uses svn (subversion) rather than git to get GSAS-II updates. I am recommending that everyone using the gsas2full installer reinstall from GitHub following these directions. I am expecting that after September, this will be needed to get GSAS-II updates.

**Comment by briantoby** [comment]
I really don't know what went wrong for you. I can see that the fit you did failed to progress very far (the scale factor only went to 1.5 not 50). 

When I tried running the 1st few steps of the tutorial, they worked fine for me. Also, when I tried refining with your .gpx file, the refinement also worked fine and dropped to Rwp of 45.2% as predicted for step 5, so I am at a loss. 

Please update and try again. If you can reproduce this again, I'd really like to see what you are doing via Zoom or Teams.

**Comment by spinglery** [comment]
Dear Brian, 
Thanks for your helpful comments. I had installed full version 5786 and then prior to bug reporting ;-) updated to version 5793. I had observed the odd refinements with the highly correlated parameters both in version 5786 and the updated version 5793. Now with a fresh full install of version 5793, the refinements proceed as expected without any highly correlated parameters.
Best regards
Bernhard
